sci-electronics/librepcb/librepcb-1.0.0.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 2009-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake virtualx xdg
+
+DESCRIPTION="Free EDA software to develop printed circuit boards"
+HOMEPAGE="https://librepcb.org/"
+SRC_URI="https://download.librepcb.org/releases/${PV}/${P}-source.zip"
+SRC_URI="https://download.librepcb.org/releases/${PV/_/-}/librepcb-${PV/_/-}-source.zip"
```

Comment:
Why do we overwrite `SRC_URI` here? Should we drop the first `SRC_URI=` since this is now not doing anything?

Reply:
Yes, first SRC_URI must be dropped, it's my fail.